packages/expo-polyfills/package.json

```diff
@@ -30,6 +30,7 @@
   "dependencies": {
     "base-64": "1.0.0",
     "expo-crypto": "12.6.0",
+    "expo-secure-store": "12.3.1",
```

Comment:
im wondering if this (and maybe some/all the other dependencies) should be peer deps? this version will quickly become desynced with expo's ~quarterly releases.